index.html

```diff
@@ -78,5 +78,243 @@ <h2><a href="#">LET’S CONNECT</a></h2>
+                        </ul>
+                    </div>
+                    <div class="buttons-wrapper">
+                        <button type="button" class="buttons">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- About Section -->
+    <section class="about">
+        <div class="A-title">
+            <h2>About <br> Myself</h2>
+        </div>
+        <div class="info">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+        </div>
+
+        <div class="social-media">
+            <h2><a href="#">LET’S CONNECT</a></h2>
+        </div>
+        
+        <ul class="social-icons">
+            <li>
+                <a href="https://twitter.com/dok_theo">
+                    <img src="images/twitter.png" alt="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="https://linkedin.com/in/Theophilusdoku">
+                    <img src="images/linkedin.png" alt="linkedin">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="images/medium.png" alt="medium">
+                </a>
+            </li>
+            <li>
+                <a href="https://github.com/theodoku">
+                    <img src="images/github.png" alt="github">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="images/angellist.png" alt="angellist">
+                </a>
+            </li>
+        </ul>
+
+        <div class="button-about">
+            <a href="#" class="btn">Get my resume</a>
+        </div>
+
+        <ul class="programming-skills">
+            <li class="languages">
+              <h3 class="langTitle">Languages</h3>
+              <div class="drop">
+                <img src="images/drop.png" id="drop"   alt="dropdown" />
+              </div>
+            </li>
+            <ul class="languages-list">
+              <li class="lang-skills">
+                <div class="langImg">
+                  <img src="images/js.png" id="langImag" alt="javascript" />
+                </div>
+                <h4>Javascript</h4>
+              </li>
+              <li class="lang-skills">
+                <div class="langImg">
+                  <img src="images/html.png" id="langImag" alt="html" />
+                </div>
+                <h4>HTML</h4>
+              </li>
+              <li class="lang-skills">
+                <div class="langImg">
+                  <img src="images/css.png" id="langImag" alt="css" />
+                </div>
+                <h4>CSS</h4>
+              </li>
+            </ul>
```

Comment:
:red_circle: For semantics reason, Kindly make sure you wrap this block in a li tag as only li tag should be direct child of ul tag.